solr/docker/build.gradle

```diff
@@ -0,0 +1,95 @@
+import org.gradle.tooling.model.SourceDirectory
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  id "base"
+  id "com.palantir.docker" version "0.25.0"
+}
+
+subprojects {
+  apply plugin: 'base'
+  apply plugin: "com.palantir.docker"
+}
+
+description = 'Solr Docker image'
+
+docker {
+  name = "apache/solr:${version}"
+  files file('include'), '.dockerignore'
+  buildArgs(['BASE_IMAGE' : 'openjdk:11-jre-slim', 'SOLR_PACKAGE_IMAGE' : 'apache/solr-build:local-package', 'SOLR_VERSION': "${version}"])
+}
+
+// Create the docker image as a part of the assemble task
+assemble.dependsOn tasks.docker
```

Comment:
Hi,
as noted on Slack already. Like for the refguide, don't yet execute docker by default on assemble.
I would make it optional for now, until we find a way how to assemble docker without docker or make it print a warning if no docker is enabled.

Reply:
+1 100% to what Uwe said